doc/source/citiesapps_com.md

```diff
@@ -0,0 +1,233 @@
+# App CITIES
+
+Support for schedules provided by [App CITIES](https://citiesapps.com), serving App CITIES, Austria.
+
+## Supported cities
+
+|City|Website|
+|-|-|<!--Begin of service section-->
```

Comment:
I assume that the github markdown renderer will not like the comment within the table. Didn't you fix a similar problem some weeks ago?